package.json

```
@@ -2,7 +2,7 @@
   "name": "@google/gemini-cli",
   "version": "0.1.9",
   "engines": {
-    "node": ">=18.0.0"
+    "node": ">=20.0.0"
```

```suggestion
    "node": ">=20.19.0"
```

According to the change in `CONTRIBUTING.md`

---

Would it be possible to switch to 20.18.x? Being that as mentioned on some Linux `20.18` operating systems it is the default version and could generate some warnings.